starters/nuxt-2/store/checkout.js

```diff
@@ -0,0 +1,34 @@
+import { get, set, del } from 'idb-keyval';
+import createShopifyCheckoutClient from '@nacelle/shopify-checkout';
+
+export const actions = {
+  async initCheckout({ commit }) {
+    try {
+      const checkoutClient = createShopifyCheckoutClient(this.$config.shopify);
+      const checkoutId = await get('checkoutId');
+      if (checkoutId) {
+        const checkout = checkoutClient.get({
+          id: checkoutId
+        });
+        if (checkout?.completed) {
+          await del('checkoutId');
+          commit('cart/clearCart', false, { root: true });
+        }
+      }
+    } catch {}
```

Comment:
Does this really need to be in a `try/catch`? If so, let's do something in the `catch`.